--- libcap-1.10/libcap/include/sys/capability.h.foo	Fri Nov  9 16:26:25 2001
+++ libcap-1.10/libcap/include/sys/capability.h	Fri Nov  9 16:28:47 2001
@@ -21,6 +21,16 @@
  */
 
 #include <sys/types.h>
+#include <stdint.h>
+
+/*
+ * Make sure we can be included from userland by preventing
+ * capability.h from including other kernel headers
+ */
+#define _LINUX_TYPES_H
+#define _LINUX_FS_H
+typedef unsigned int __u32;
+
 #include <linux/capability.h>
 
 /*
